src/paradigma/tremor/tremor_analysis_config.py

```diff
@@ -0,0 +1,97 @@
+from typing import Dict, List
+
+from paradigma.constants import DataColumns, DataUnits
+
+class IMUConfig:
+    """
+    Base class for Tremor feature extraction and Tremor detection configurations, based on the IMU data (accelerometer, gyroscope).
+    """
+    def __init__(self):
+
+        self.time_colname = DataColumns.TIME
+
+        self.l_accelerometer_cols: List[str] = [
+            DataColumns.ACCELEROMETER_X,
+            DataColumns.ACCELEROMETER_Y,
+            DataColumns.ACCELEROMETER_Z,
+        ]
+        self.l_gyroscope_cols: List[str] = [
+            DataColumns.GYROSCOPE_X,
+            DataColumns.GYROSCOPE_Y,
+            DataColumns.GYROSCOPE_Z,
+        ]
+
+        self.l_gravity_cols: List[str] = [
+            DataColumns.GRAV_ACCELEROMETER_X,
+            DataColumns.GRAV_ACCELEROMETER_Y,
+            DataColumns.GRAV_ACCELEROMETER_Z,
+        ]
+
+    def set_sensor(self, sensor: str) -> None:
+        """Sets the sensor and derived filenames"""
+        self.sensor: str = sensor
+        self.set_filenames(sensor)
+
+    def set_filenames(self, prefix: str) -> None:
+        """Sets the filenames based on the prefix,
+        
+        Parameters
+        ----------
+        prefix : str
+            The prefix for the filenames.
+        """
+        self.meta_filename = f"{prefix}_meta.json"
+        self.time_filename = f"{prefix}_time.bin"
+        self.values_filename = f"{prefix}_samples.bin"
```

Comment:
Een (terechte) comment vanuit eScience was dat we de rauwe data "samples" noemen en als ze processed zijn "values". Kijken we volgende conversie wel een keer naar, maar dan ben je ook op de hoogte.